perf/benchmark/README.md

```diff
@@ -86,77 +86,81 @@ Required fields:
   --no-clientsidecar  do not run clientsidecar and serversidecar for all
-  --ingress INGRESS   run traffic thru ingress
+  --ingress INGRESS   run traffic through ingress
   --labels LABELS     extra labels
 ```
 
 
-### Example 1 
+### Example 1
 
 `runner.py` will run all combinations of the parameters given. For example:
 
 
 ```bash
-python runner/runner.py 1,2,4,8,16,32,64 1000 240 --serversidecar 
+python runner/runner.py 1,2,4,8,16,32,64 1000 240 --serversidecar
 ```
 
-- This will run separate tests for the `both` and `serversidecar` modes 
-- Separate tests for 1 to 64 concurrent connections 
-- Each connection will send **1000** QPS 
+- This will run separate tests for the `both` and `serversidecar` modes
+- Separate tests for 1 to 64 concurrent connections
+- Each connection will send **1000** QPS
 - Each test will run for **240** seconds
 
-### Example 2 
+### Example 2
 
 ```bash
 python runner/runner.py 16,64 1000,4000 180 --serversidecar --baseline
 ```
 
-- 12 tests total, each for **180** seconds, with all combinations of: 
-- **16** and **64** connections 
-- **1000** and **4000** QPS 
-- `both`,  `serversidecar`, and `baseline` proxy modes 
+- 12 tests total, each for **180** seconds, with all combinations of:
+- **16** and **64** connections
+- **1000** and **4000** QPS
+- `both`,  `serversidecar`, and `baseline` proxy modes
 
 
-## [Optional] Disable Mixer 
+## [Optional] Disable Mixer
 
 Calls to Istio's Mixer component (policy and telemetry) adds latency to the sidecar proxy. To disable Istio's mixer and re-run the performance tests:
 
 
-1. Disable Mixer 
+1. Disable Mixer
 
-```bash 
+```bash
 kubectl -n istio-system get cm istio -o yaml > /tmp/meshconfig.yaml
 python ../../bin/update_mesh_config.py disable_mixer /tmp/meshconfig.yaml | kubectl -n istio-system apply -f /tmp/meshconfig.yaml
 ```
 
 2. Run `runner.py`, in any sidecar mode, with the `--labels=nomixer` flag.
 
-3. Re-enable Mixer: 
+3. Re-enable Mixer:
 
-```bash 
+```bash
 kubectl -n istio-system get cm istio -o yaml > /tmp/meshconfig.yaml
 python ../../bin/update_mesh_config.py enable_mixer /tmp/meshconfig.yaml  | kubectl -n istio-system apply -
 ```
 
-## Gather Result Metrics 
+## Gather Result Metrics
 
-Once `runner.py` has completed, extract the results from Fortio and Prometheus. 
+Once `runner.py` has completed, extract the results from Fortio and Prometheus.
 
-1. Set `FORTIO_CLIENT_URL` to the `fortioclient` Service's `EXTERNAL_IP`: 
+1. Set `FORTIO_CLIENT_URL` to the `fortioclient` Service's `EXTERNAL_IP`:
 
 ```bash
-kubectl get svc -n $NAMESPACE fortioclient
+$ kubectl get svc -n $NAMESPACE fortioclient
+NAME           TYPE           CLUSTER-IP   EXTERNAL-IP     PORT(S)                                                       AGE
+fortioclient   LoadBalancer   10.0.28.56   35.225.105.18   8080:31759/TCP,8079:30495/TCP,8078:31107/TCP,8077:31034/TCP   16h
+
+$ export FORTIO_CLIENT_URL=http://35.225.105.18:8080
```

Comment:
probably don't want to commit your public ip address here.

We could replace with 

`export FORTIO_CLIENT_URL=$(kubectl get service fortioclient -o jsonpath='{.status.loadBalancer.ingress[0].ip}'):8080`

maybe